deploy/eck-resources/examples/elasticsearch/detailed.yaml

```diff
@@ -0,0 +1,458 @@
+# Global settings that will be applied to all resources (Elasticsearch/Kibana/etc)
+#
+global:
```

Comment:
`global` is a keyword for values that can be accessed by all charts and subcharts (https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#global-chart-values). I don't think of any side effect that could happen by re-using it for a different purpose here for now, but better check twice or use a different value name.